app/src/lib/email.ts

```diff
@@ -44,7 +44,7 @@ export function lookupPreferredEmail(account: Account): string {
  */
 function isEmailPublic(email: IAPIEmail): boolean {
   // If an email doesn't have a visibility setting it means it's coming from an
-  // older Enterprise Server which doesn't have the concept of visibility.
+  // older Enterprise which doesn't have the concept of visibility.
```

Comment:
Just to avoid any confusion for a reader let's make it a bit more explicit that it's not "an enterprise's email" but the version of Enterprise that they were running.

```suggestion
  // older Enterprise version which doesn't have the concept of visibility.
```

Reply:
Makes total sense to me, thanks!